partOfSpeech.ts

```diff
@@ -0,0 +1,217 @@
+import { hasProp } from './util';
+
+const clauses = {
+  'n..': 'noun',
+  'n.':  'noun',
+  n:     'noun',
+
+  'a. .':   'adjective',
+  'a.':     'adjective',
+  a:        'adjective',
+  'p. a.':  'past adjective', // ?? ?? ??
+  'p.a.':   'past adjective', //  ?? ?? ??
+  'pr. a.': 'proper adjective',
+  'adj.':   'adjective', // but why also just 'a.'?
+
+  'v.':    'verb',
+  'v. i.': 'intransitive verb',
+  'v. t.': 'transitive verb',
+
+  'imp.':       'imperfect (simple past tense)', // could also be imperative
```

Comment:
I think the additional explaination in the braces (simple past tense) can be left out. It seems to be used in the form imp. & p. p. a lot.